qa/vagrant-helpers.rb

```diff
@@ -1,15 +1,26 @@
 # encoding: utf-8
 require "open3"
+require "json"
 
 module LogStash
   class VagrantHelpers
+    DEFAULT_CENTOS_BOXES = [ "centos-6", "centos-7" ]
+    DEFAULT_DEBIAN_BOXES = [ "ubuntu-1204", "ubuntu-1404"]
 
-    def self.bootstrap
-      execute("vagrant up")
+    def self.bootstrap(box)
+      Dir.chdir("acceptance") do
+        p execute("vagrant up #{box} --provider virtualbox")
```

Comment:
major refactoring needs to happen here. but the `--provider virtualbox` should stay. some environments have docker as the provider, or something else as the default.